dev/scripts/reporting-functions.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
This file seems to share a lot of content with `scripts/reporting/reporting-functions.sh`. Do we need both?